openpgp/openpgp.d.ts

```diff
@@ -377,7 +384,18 @@ declare module openpgp.key {
      */
     interface Key {
         armor(): String,
-        decrypt(passphrase: String): Boolean,
+        decrypt(passphrase: string): boolean;
+        getExpirationTime(): Date;
+        getKeyIds(): Array<any>;
+        getPreferredHashAlgorithm(): string;
+        getPrimaryUser(): any;
+        getUserIds(): Array<string>;
+        isPrivate(): boolean;
+        isPublic(): boolean;
+        primaryKey: packet.PublicKey;
```

Comment:
You have probably noticed that openpgp's documentations generally sucks. The primaryKey property is instrumental, yet they failed to add any documentation on it.